cristin-project/src/test/java/no/unit/nva/cristin/projects/fetch/project.feature

```diff
@@ -167,3 +167,11 @@ Feature: API tests for Cristin Project retrieve and search
     And match response.approvals[0].applicationCode == 'EthicalApproval'
     And match response.approvals[0].identifier == '2017/800'
     And match response.approvals[0].authorityName == '#present'
+
+  Scenario: Fetch returns project with webpage
```

Comment:
You might want the non-webpage test too

Reply:
We might need to decide if field should be null or just omitted first, but i can always add a test now and change it later